components/brave_ads/core/internal/database/database.cc

```diff
@@ -15,12 +15,20 @@
 #include "base/functional/bind.h"
 #include "brave/components/brave_ads/core/internal/common/database/database_bind_util.h"
 #include "brave/components/brave_ads/core/internal/common/database/database_record_util.h"
+#include "brave/components/brave_ads/core/internal/legacy_migration/database/database_constants.h"
 #include "sql/meta_table.h"
 #include "sql/statement.h"
 #include "sql/transaction.h"
 
 namespace brave_ads {
 
+namespace {
+
+constexpr char kCreativeAdNotificationsTable[] = "creative_ad_notifications";
```

Comment:
As discussed `geo_targets` maybe better

Reply:
Discussed using `SELECT count(*) FROM sqlite_master WHERE type='table' AND name<>'meta'`